sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -98,7 +129,7 @@ def span_instance(self) -> Span:
 
     def span(self, name: str = "span", **kwargs: Any) -> "OpenTelemetrySpan":
         """
-        Create a child span for the current span and append it to the child spans list in the span instance.
+        Create a child span for the current span and return it.
```

Comment:
This docstring was actually misleading because there isn't a "child spans list" that is/was appended to. I just updated the docstring to be more accurate.